packages/server/.gitignore

```diff
@@ -1,3 +1,4 @@
+lib/browsers/cri-client.js
```

Comment:
This shouldn't be necessary.

In fact the other `lib/util/*` files ignored here should also be removed.

We build the `.ts` files at runtime, and only transpile them when building the binary.

Reply:
removed, added a note to .gitignore